extensions/amp-story/1.0/amp-story-page.js

```diff
@@ -1853,6 +1854,20 @@ export class AmpStoryPage extends AMP.BaseElement {
     }
   }
 
+  /**
+   * Adds an empty alt tag to amp-img elements if not present.
+   * Prevents screen readers from announcing the img src value.
+   * @private
+   */
+  initializeImgAltTags_() {
+    Array.prototype.forEach.call(
+      this.element.querySelectorAll('amp-img'),
```

Comment:
To get this right and avoid all race conditions, we need to:
1. if amp-img doesn't have an img child, add the empty alt
2. if amp-img has an img-child, use the `propagateAttributes` helper (inherited from `base-element.js`) on `amp-img` to propagate the change you made on amp-img to its img